### ISSUE TITLE
Fix type errors caused by external libraries

### ISSUE THREAD

**Issue by senofsky** [senofsky/face-mask-detector#11]
The following is a snippet of received type errors:

```
face_mask_detector/lib.py:4: error: Cannot find implementation or library stub for module named 'cv2'
face_mask_detector/lib.py:6: error: Cannot find implementation or library stub for module named 'imutils'
face_mask_detector/lib.py:7: error: Skipping analyzing 'numpy': found module but no type hints or library stubs
face_mask_detector/lib.py:11: error: Cannot find implementation or library stub for module named 'imutils.video'
face_mask_detector/lib.py:12: error: Cannot find implementation or library stub for module named 'tensorflow.keras.applications.mobilenet_v2'
face_mask_detector/lib.py:13: error: Cannot find implementation or library stub for module named 'tensorflow.keras.preprocessing.image'
face_mask_detector/lib.py:14: error: Cannot find implementation or library stub for module named 'tensorflow.keras.models'
```

[This resource](https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports) may be of use.
